src/newssourceComponent.js

```diff
@@ -0,0 +1,16 @@
+import errorPopupHandler from "./errorHandler.js";
+
+export default class ArticlesFetcher {
```

Comment:
Do you still need ArticlesFetcher if you have Api class?

Reply:
I am not using I will delete